UC-Berkeley-Extension/global
# Switch to component module

## Asana Ticket(s)
- Angular Components

## Description
An attempt at setting up the system to build angular components using a different module. 

## Testing instructions
1. `composer install`
1. Build the site with `phing install` from the `/www/html` folder
1. Confirm that the [Modules List](https://global.ddev.site/admin/modules) no longer has PDB and now has Component
1. Create a new node using the Layout Builder CT
1. Add the three Angular components: 
    * Voices Spotlight
    * Voices List
    * Info Sessions List
1. Verify they all show up something like this:
![image](https://user-images.githubusercontent.com/17348237/97970446-6606cc00-1d76-11eb-9c31-8cabb40616ad.png)


## Remaining tasks
- [x] Create simple component
- [x] Recreate existing components
    - [x] Voices List
    - [x] Voices Spotlight
    - [x] Info Sessions List
- [x] Ensure error handling
